index.js

```diff
@@ -55,6 +55,73 @@ class Vector3 {
 	}
 }
 
+class STLMeasures {
+  constructor() {
+  	this.volume = 0;
+  	this.area = 0;
+  	this.minx = Infinity;
+  	this.maxx = -Infinity;
+  	this.miny = Infinity;
+  	this.maxy = -Infinity;
+  	this.minz = Infinity;
+  	this.maxz = -Infinity;
+    this.xCenter = 0,
+    this.yCenter = 0,
+    this.zCenter = 0;
```

Comment:
The `,` seem a bit unusual at this place. I assume the V8 will actually like it but probably the pre-optimisation should handle this. I can see a potential misuse here.

Reply:
Sorry, this was a mistake, a copy&paste problem.